<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Expense Tracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>511</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="income_expense">
     <widget class="QLabel" name="amount_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Amount:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_record_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Record</string>
      </property>
     </widget>
     <widget class="QLabel" name="necessity_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Necessity:</string>
      </property>
     </widget>
     <widget class="QLabel" name="date_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
     <widget class="QLabel" name="bank_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>58</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Bank:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="bank_choose_field">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="amount_entry_field">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Toman</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_selector_income_expense">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>M/d/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="sql_tableview">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>60</y>
        <width>351</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="necessity_choose_field">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>330</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Yes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>No</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="back_to_summary_income_expense_button">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>460</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back To Summary</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_record_income_expense_button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>460</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Record</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bank_account">
     <widget class="QLabel" name="apr_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>APR Rate:</string>
      </property>
     </widget>
     <widget class="QLabel" name="date_label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
     <widget class="QLabel" name="deposit_amount_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Deposit Amount:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_balances_button">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>470</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Balances</string>
      </property>
     </widget>
     <widget class="QLabel" name="gets_apr_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Gets APR?</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="bank_name_entry_field">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Melli, Mellat, Blu, etc..</string>
      </property>
     </widget>
     <widget class="QComboBox" name="apr_choose_field">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>340</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <item>
       <property name="text">
        <string>Monthly</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Annually</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="apr_type_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>APR Type:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="deposit_amount_entry_field">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Toman</string>
      </property>
     </widget>
     <widget class="QLabel" name="days_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>410</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Days:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="days_spinbox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>410</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>365</number>
      </property>
     </widget>
     <widget class="QPushButton" name="add_bank_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Bank</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_selector_bank">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
      <property name="displayFormat">
       <string>M/d/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="bank_name_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bank Name:</string>
      </property>
     </widget>
     <widget class="QTableView" name="bank_account_tableView">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>321</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="apr_rate_entry_field">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>290</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_to_summary_bank_account_button">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>470</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back to Summary</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="gets_apr_group">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: none;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="yesradio">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>yes</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="noradio">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>no</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="delete_record_bank_button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>470</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Record</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="summary">
     <widget class="QLabel" name="summary_label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>36</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Summary</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="summary_container" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>721</width>
        <height>271</height>
       </rect>
      </property>
      <widget class="QWidget" name="total_necessary_spending_box" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>151</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget { background-color: rgba(210, 255, 210, 180); 
border: 1px solid rgba(120, 200, 120, 200); 
border-radius: 8px; } 
</string>
       </property>
       <widget class="QLabel" name="total_necessary_spending_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>131</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #2E7D32;
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total &lt;/p&gt;&lt;p&gt;Necessary &lt;/p&gt;&lt;p&gt;Spending&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="total_necessary_spending_number_output">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>189</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #1B5E20;
}</string>
        </property>
        <property name="text">
         <string>$0.00 (PlaceHolder)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="total_unnecessary_spending_box" native="true">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>0</y>
         <width>161</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget { background-color: rgba(255, 220, 220, 180); 
border: 1px solid rgba(220, 150, 150, 200); 
border-radius: 8px; } 
</string>
       </property>
       <widget class="QLabel" name="total_unnecessary_spendingnumber_holder">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>190</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #C62828;
}</string>
        </property>
        <property name="text">
         <string>$0.00 (PlaceHolder)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="total_unnecessary_spending_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>141</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #B71C1C;
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total &lt;/p&gt;&lt;p&gt;Unnecessary &lt;/p&gt;&lt;p&gt;Spending&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="total_revenue_box" native="true">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>0</y>
         <width>161</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget { background-color: rgba(210, 235, 255, 180); 
border: 1px solid rgba(140, 180, 220, 200);
border-radius: 8px; } 
</string>
       </property>
       <widget class="QLabel" name="total_revenue_number_output">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>189</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #0D47A1;
}</string>
        </property>
        <property name="text">
         <string>$0.00 (PlaceHolder)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="total_revenue_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>141</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #1565C0;
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total &lt;/p&gt;&lt;p&gt;Revenue&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="total_spending_box" native="true">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>0</y>
         <width>161</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget { background-color: rgba(240, 240, 240, 170); 
border: 1px solid rgba(200, 200, 200, 200); 
border-radius: 8px; } 
</string>
       </property>
       <widget class="QLabel" name="total_spending_number_holder">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>189</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #212121;
}</string>
        </property>
        <property name="text">
         <string>$0.00 (PlaceHolder)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="total_spending_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>141</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #424242;
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total&lt;/p&gt;&lt;p&gt;Spending&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="add_income_expense_button">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>480</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Income/Expense</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_bank_account_button">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>480</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Bank Account</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
